core/platform/email/gae_email_services.py

```diff
@@ -22,29 +22,6 @@
 from google.appengine.api import app_identity
 from google.appengine.api import mail
 
-
-def send_mail_to_admin(subject, body):
-    """Enqueues a 'send email' request with the GAE mail service.
-
-    Args:
-      - subject: str. The subject line of the email.
-      - body: str. The plaintext body of the email.
-    """
-    if feconf.CAN_SEND_EMAILS_TO_ADMIN:
-        if not mail.is_email_valid(feconf.ADMIN_EMAIL_ADDRESS):
-            raise Exception(
-                'Malformed email address: %s' %
-                feconf.ADMIN_EMAIL_ADDRESS)
-
-        app_id = app_identity.get_application_id()
```

Comment:
Actually, sorry, no. This should go in the specific email body in cron.py, I think. Sometimes emails are sent to the admin in their role as admin, but you could also have generic emails sent to the admin as a regular user. So I don't think we want to prepend this to every single email sent to the admin...